### DISCUSSION TITLE
Old forum posts.

### DISCUSSION THREAD

**Discussion by exenestecnico** [adonisjs/core#1255] [General]
Will the posts from the old forum be migrated to here? If not, please don't shut down the old forum.

**Comment by popovicieduard** [comment]
We should find a way migrating the old forum posts, there is a lot of knowledge out there and it would be a shame to lose it all. I don't know if there is a migration function for github but I could give a hand in scraping the old forum and provide a csv.

**Comment by thetutlage** [comment]
I agree, that is lot of knowledge in the forum posts, but just dumping them here also will add a lot of noise and also, we need to handle a couple of things

- How to link people involved in the thread with the Github discussions? The accounts created on the Forum may not have used the same email address as it is on Github and hence their will be orphan threads and comments.

- Also, someone will have to manually go to individual thread and mark them as answered. But this the original author is not marking them as answered.

Any good suggestions on how to handle this?

**Comment by RomainLanz** [comment]
I believe we should keep it open (in read-only, like right now) for a while. When this forum (Github Discussion) will have more content we can freely close it.
